Cargo.toml

```diff
@@ -21,9 +21,9 @@ arbitrary = { version = "1.0.1", features = ["derive"], optional = true }
 impl-trait-for-tuples = "0.2.2"
 
 [dev-dependencies]
-criterion = "0.3.0"
+criterion = "0.4.0"
 serde_derive = { version = "1.0" }
-parity-scale-codec-derive = { path = "derive", default-features = false }
+parity-scale-codec-derive = { version = ">= 3.5.0", path = "derive", default-features = false }
```

Comment:
Do you need to specify version req?

Reply:
Technically I don't, but I like to guarantee that the new derive crate is used if the main crate's bumped. Especially in cases like this where the bug that I'm fixing could in theory have security implications and the changes in the derive crate are necessary to fix it. We don't want someone to only bump the main crate and accidentally forget about bumping the derive crate and then have a surprise pikachu moment. (For [my own serialization crate I just always force a specific version of the derive crate](https://github.com/koute/speedy/blob/master/Cargo.toml#L17) so there's never no funny business going on.)

(Besides, we don't really test mismatched versions of those two crates, so in theory there could be other obscure bugs popping up if someone uses them like that.)